envs/example/ci-full/group_vars/all.yml

```diff
@@ -1,17 +1,33 @@
 ---
-stack_env: example_ci-full
+stack_env: ci-full
+ursula_os: 'ubuntu'
+undercloud_cidr:
+  - cidr: 192.168.0.0/22
 
 state_path_base: /opt/stack/data
 
-neutron:
-  enable_external_interface: True
-  lbaas:
-    enabled: True
+
+hostname_fqdn: "{{ ansible_fqdn }}"
+hostname_short: "{{ hostname_fqdn.split('.')|first }}"
+
+floating_ip: "{{ hostvars[groups['controller'][0]][primary_interface]['ipv4']['address'] }}"
```

Comment:
You are right,  hostname_fqdn and hostname_fqdn are not needed in 31x. In addition, floating_ip  refers to controller_primary which is defined in test/setup. I replaced it as I didn't run test/setup.

~~~
floating_ip: "{{ controller_primary }}"
~~~